docs/build.sh

```diff
@@ -161,4 +161,8 @@ rsync -av \
 # To speed up validate_fragment invocations in validating_code_block
 bazel build "${BAZEL_BUILD_OPTIONS[@]}" //tools/config_validation:validate_fragment
 
+curl -sf https://gobinaries.com/cluttercode/clutter/cmd/clutter@0.2.1 | sh
```

Comment:
hi @itayd to make this work you would need to move this intall outside of the python venv - eg by doing it in  `do_ci.sh`

Reply:
...or higher in the file before it enters the venv

Reply:
hmm, on closer inspection it seems that permissions are dropped in `run_envoy_docker.sh` - which iirc means you need to add it into the envoy build image - here https://github.com/envoyproxy/envoy-build-tools

Reply:
or avoid root installing somehow i guess

Reply:
Yeah, I think I prefer adding it to the base image or using some Bazel Go package stuff, rather than picking up a rando binary via curl.

Reply:
i'm giving it a go at https://github.com/envoyproxy/envoy-build-tools/pull/124

Reply:
The only thing I'd add is that we do want to allow a local docs build that doesn't have this image (i.e. `./docs/build.sh`).

We would need instructions on how to install Clutter locally in https://github.com/envoyproxy/envoy/blob/main/bazel/README.md. But then we might get in trouble if Clutter needs upgrading as you develop.

I'm tempted to say the solution is Bazel Go specification of the dependency here. Then running the binary under `bazel run`. Something like that?